packages/fury/lib/fury.js

```diff
@@ -75,12 +74,16 @@ class Fury {
       options = Object.assign(options, adapterOptions);
     }
 
-    return adapter.validate(options, (err, elements) => {
-      if (!elements || elements instanceof this.minim.Element) {
-        done(err, elements);
-      } else {
-        done(err, this.load(elements));
+    return adapter.validate(options, (err, parseResult) => {
+      if (!parseResult) {
```

Comment:
Shouldn't this be `if (err) {`?

Reply:
I was protecting against older adapters which behave incorrectly, if we have an adapter that returns both an error and a parseResult, fury will discard the error